manager/src/grype_db_manager/distribution.py

```diff
@@ -17,8 +17,8 @@
     from collections.abc import Generator
 
 DB_SUFFIXES = {".tar.gz", ".tar.zst"}
-MAX_DB_AGE = 120  # ~4 months in days
-MINIMUM_DB_COUNT = MAX_DB_AGE  # number of db entries per schema
+MAX_DB_AGE = 7  # old db listings are making the file large and slowing download times
```

Comment:
My vote is to make this 2, which would support what I think are the main use cases:
* get the latest
* see what's changed in the latest

Reply:
Sounds good. I'll do that and also figure out why unit tests are failing.